config/environments/pentest.rb

```diff
@@ -88,23 +88,50 @@
     config.logger    = ActiveSupport::TaggedLogging.new(logger)
   end
 
+  # Mitigate X-Forwarded-Host injection attacks
+  config.action_controller.default_url_options = { :host => ENV['PROD_HOSTNAME'], protocol: 'https' }
+  config.action_controller.asset_host = ENV['PROD_HOSTNAME']
+
   # Mailer settings
   config.action_mailer.default_url_options = { :host => ENV['PROD_HOSTNAME'], protocol: 'https' }
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.perform_deliveries = true
   config.action_mailer.raise_delivery_errors = true
   config.action_mailer.smtp_settings = {
-      address:              'smtp.sendgrid.net',
-      port:                 2525,
-      user_name:            ENV['SENDGRID_USERNAME'],
-      password:             ENV['SENDGRID_PASSWORD'],
-      domain:               ENV['PROD_HOSTNAME'],
-      authentication:       'plain',
-      enable_starttls_auto: true
+    address:              'smtp.sendgrid.net',
+    port:                 2525,
+    user_name:            ENV['SENDGRID_USERNAME'],
+    password:             ENV['SENDGRID_PASSWORD'],
+    domain:               ENV['PROD_HOSTNAME'],
+    authentication:       'plain',
+    enable_starttls_auto: true
   }
 
   # CUSTOM CONFIGURATION
 
   # disable admin notification (like startup email)
   config.disable_admin_notifications = true
+
+  config.bard_host_url = 'https://terra-bard-dev.appspot.com'
+
+  # Inserts middleware to perform automatic connection switching.
+  # The `database_selector` hash is used to pass options to the DatabaseSelector
+  # middleware. The `delay` is used to determine how long to wait after a write
+  # to send a subsequent read to the primary.
+  #
+  # The `database_resolver` class is used by the middleware to determine which
+  # database is appropriate to use based on the time delay.
+  #
+  # The `database_resolver_context` class is used by the middleware to set
+  # timestamps for the last write to the primary. The resolver uses the context
+  # class timestamps to determine how long to wait before reading from the
+  # replica.
+  #
+  # By default Rails will store a last write timestamp in the session. The
+  # DatabaseSelector middleware is designed as such you can define your own
+  # strategy for connection switching and pass that into the middleware through
+  # these configuration options.
+  # config.active_record.database_selector = { delay: 2.seconds }
+  # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
+  # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
```

Comment:
These look like inline Rails docs geared to first-time setup.  Can they be removed?

Reply:
59fbd54